app/models/force/listing.rb

```diff
@@ -0,0 +1,103 @@
+      { domain: 'lottery_venue', salesforce: 'Lottery_Venue' },
+      { domain: 'lottery_summary', salesforce: 'Lottery_Summary' },
+      { domain: 'lottery_street_address', salesforce: 'Lottery_Street_Address' },
+      { domain: 'lottery_city', salesforce: 'Lottery_City' },
+      { domain: 'lottery_url', salesforce: 'Lottery_URL' },
+      { domain: 'reserved_community_type', salesforce: 'Reserved_community_type' },
+      { domain: 'application_phone', salesforce: 'Application_Phone' },
+      { domain: 'application_organization', salesforce: 'Application_Organization' },
+      { domain: 'application_street_address', salesforce: 'Application_Street_Address' },
+      { domain: 'application_city', salesforce: 'Application_City' },
+      { domain: 'download_url', salesforce: 'Download_URL' },
+      { domain: 'application_state', salesforce: 'Application_State' },
+      { domain: 'application_postal_code', salesforce: 'Application_Postal_Code' },
+      { domain: 'in_lottery', salesforce: 'In_Lottery' },
+      { domain: 'leasing_agent_name', salesforce: 'Leasing_Agent_Name' },
+      { domain: 'leasing_agent_title', salesforce: 'Leasing_Agent_Title' },
+      { domain: 'leasing_agent_email', salesforce: 'Leasing_Agent_Email' },
+      { domain: 'leasing_agent_phone', salesforce: 'Leasing_Agent_Phone' },
+      { domain: 'legal_disclaimers', salesforce: 'Legal_Disclaimers' },
+      { domain: 'building_selection_criteria', salesforce: 'Building_Selection_Criteria' },
+      { domain: 'pet_policy', salesforce: 'Pet_Policy' },
+      { domain: 'report_id', salesforce: 'Report_id' },
+      { domain: 'required_documents', salesforce: 'Required_Documents' },
+      { domain: 'smoking_policy', salesforce: 'Smoking_Policy' },
+      { domain: 'eviction_history', salesforce: 'Eviction_History' },
+      { domain: 'criminal_history', salesforce: 'Criminal_History' },
+      { domain: 'credit_rating', salesforce: 'Credit_Rating' },
+      { domain: 'lottery_status', salesforce: 'Lottery_Status' },
+      { domain: 'office_hours', salesforce: 'Office_Hours' },
+      { domain: 'information_sessions', salesforce: 'Information_Sessions' },
+      { domain: 'nflagged_applications', salesforce: 'nFlagged_Applications' },
+      { domain: 'units_available', salesforce: 'Units_Available' },
+      { domain: 'lease_signed_application', salesforce: 'Lease_Signed_Application' },
+      { domain: 'last_modified_date', salesforce: 'LastModifiedDate' },
+      { domain: 'owner', salesforce: 'Owner' },
+      { domain: 'account', salesforce: 'Account' },
+      { domain: 'building', salesforce: 'Building' },
+      { domain: 'listing_lottery_preferences', salesforce: 'Listing_Lottery_Preferences' },
+      { domain: 'units', salesforce: 'Units' },
+    ].freeze
+
+    def to_domain
+      domain_fields = super
+      return if domain_fields.blank?
+
+      domain_fields[:is_sale] = Force::ListingService.sale?(domain_fields)
```

Comment:
did we have these properties in mapListing before?

Reply:
we did, but they were added in listing service and this feels like a better place to put them.